### PR TITLE
[CI] added dependabot config

### PR DESCRIPTION
**Pull request by mryzhov**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      timezone: "Europe/Dublin"
+    open-pull-requests-limit: 3
+    assignees:
+      - "apaniukov"
+    versioning-strategy: increase-if-necessary
+  #
+  # Tests
+  #
+  - package-ecosystem: pip
+    directory: "/tests"
```
Comment (ilya-lavrenov) [inline .github/dependabot.yml:22]: I don't see requirements files in `tests` folder

Comment (mryzhov) [reply .github/dependabot.yml:22]: yes, but they could be there in future...

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# See help here: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/enabling-and-disabling-version-updates
+
+version: 2
+updates:
+  #
+  # Python dependencies
+  #
+  - package-ecosystem: pip
+    directory: "/python"
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply .github/dependabot.yml]: removed

File: .github/dependabot.yml (hunk 1 above)
Comment (mryzhov) [reply .github/dependabot.yml]: excluded such dirs

**Review by apaniukov — APPROVED** [review]

**Pull request merged**